Tita-Clone/Tita-Clone/Controllers/TabBarViewController.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  TabBarViewController.swift
+//  Tita-Clone
+//
+//  Created by 혜인 on 2021/12/11.
+//
+
+import UIKit
+
+class TabBarViewController: UITabBarController {
+    //MARK: - Properties
+    
+    //MARK: - Lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setUpTabBar()
+        }
+    
+       private func setUpTabBar() {
+           self.tabBar.tintColor = .black
+
+        
+           let mainViewController = UINavigationController(rootViewController: MainViewController())
+           mainViewController.tabBarItem.image = UIImage(named: "Tita-homeBtn")
+           mainViewController.tabBarItem.imageInsets = .init(top: 0, left: 0, bottom: -30, right: 0)
+           
+           let noticeViewController = UINavigationController(rootViewController: NoticeViewController())
+           noticeViewController.tabBarItem.image = UIImage(named: "Tita-noticeBtn")
+           noticeViewController.tabBarItem.imageInsets = .init(top: 0, left: 0, bottom: -30, right: 0)
+
+           let alarmViewController = UINavigationController(rootViewController: AlarmViewController())
+           alarmViewController.tabBarItem.image = UIImage(named: "Tita-alarmBtn")
+           alarmViewController.tabBarItem.imageInsets = .init(top: 0, left: 0, bottom: -30, right: 0)
+           
+           let mypageViewController = UINavigationController(rootViewController: MypageViewController())
+           mypageViewController.tabBarItem.image = UIImage(named: "Tita-mypageBtn")
+           mypageViewController.tabBarItem.imageInsets = .init(top: 0, left: 0, bottom: -30, right: 0)
+           
+           viewControllers = [mainViewController, noticeViewController, alarmViewController, mypageViewController]
+    }
```

Comment:
```suggestion
           viewControllers = [mainViewController, noticeViewController, alarmViewController, mypageViewController]
           self.tabBar.isTranslucent = false
           self.tabBar.backgroundColor = .white
    }
```
`tabBar`가 반투명하지 않도록 속성을 설정해주는 게 좋을 것 같네요